modules/kubernetes-configuration/flux-configuration/main.bicep

```diff
@@ -63,7 +63,7 @@ resource managedCluster 'Microsoft.ContainerService/managedClusters@2022-07-01'
   name: clusterName
```

Comment:
@JPEasier found out that with the new API version, the `kustomizations` become mandatory and that's why it is failing (even without an error message). I'll implement that change later in the AVM contribution, and the same should be done here in CAMRL.

Snippet:
```bicep
kustomizations: {
  unified: {
    dependsOn: []
    force: false
    path: './cluster-manifests'
    prune: false
    syncIntervalInSeconds: 300
    timeoutInSeconds: 300
  }
}
```

Reply:
@AlexanderSehr @eriqua @JPEasier - I've updated the CARML repo to reflect the change made by @AlexanderSehr in AVM.  Could you please review.